File: src/main/java/com/catan/catanbackend/model/dto/TradeOfferMessage.java
package com.catan.catanbackend.model.dto;

import com.catan.catanbackend.model.ResourceGroup;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TradeOfferMessage {
    private String fromUser;
    private String toUser;
    private ResourceGroup offered;
    private ResourceGroup requested;
}

File: Assets/Scripts/Dtos/GameMoveResponses/TradeOfferMessage.cs
using System;
using Assets.Scripts.GameMode.Trading.Models;

namespace Assets.Scripts.Dtos.GameMoveResponses
{
    [Serializable]
    public class TradeOfferMessage
    {
        public string fromUser;
        public string toUser;
        public ResourceGroup offered;
        public ResourceGroup requested;
    }
}
